app/console.go

```diff
@@ -170,27 +228,44 @@ func createWallet(ctx *ishell.Context, app *App) {
 	ctx.Println("")
 }
 
-func createTransaction(ctx *ishell.Context, app *App) {
-	// Read in the recipient address.
-	emoji.Print(":credit_card:")
-	ctx.Println(" Enter recipient wallet address")
-	toAddress := shell.ReadLine()
-
-	// Get amount from user.
-	emoji.Print(":dollar:")
-	ctx.Println(" Enter amount to send: ")
-	amount, err := strconv.ParseUint(shell.ReadLine(), 10, 64)
-	if err != nil {
-		emoji.Println(":disappointed: Invalid number format. Please enter an amount in decimal format.")
-		return
+func editUser(ctx *ishell.Context, app *App) {
+	if len(ctx.Args) == 0 {
+		ctx.Println("Current User:")
+		ctx.Println("Name:", app.CurrentUser.Name)
+		ctx.Println("Blocksize:", app.CurrentUser.BlockSize)
+		ctx.Println("Address:", app.CurrentUser.Public().Repr())
+		emoji.Println("Emoji Address:", app.CurrentUser.Public().Emoji())
+	} else if len(ctx.Args) == 2 {
+		if ctx.Args[0] == "name" {
+			app.CurrentUser.Name = ctx.Args[1]
+			if err := app.CurrentUser.Save(userFileName); err != nil {
+				ctx.Print(err)
+			}
+			return
+		} else if ctx.Args[0] == "blocksize" {
```

Comment:
Out of scope for this PR but blocksize absolutely should not be editable by the user this way. Consensus needs to be consistent across all nodes